assets/app/view/game/alternate_corporations.rb

```diff
@@ -0,0 +1,257 @@
+# frozen_string_literal: true
+
+module View
+  module Game
+    module AlternateCorporations
+      def render_independent_mine
+        select_mine = lambda do
+          if @selectable
+            selected_corporation = selected? ? nil : @corporation
+            store(:selected_corporation, selected_corporation)
+          end
+        end
+
+        card_style = {
+          cursor: 'pointer',
+        }
+
+        card_style[:display] = @display
+
+        unless @interactive
+          factor = color_for(:bg2).to_s[1].to_i(16) > 7 ? 0.3 : 0.6
+          card_style[:backgroundColor] = convert_hex_to_rgba(color_for(:bg2), factor)
+          card_style[:border] = '1px dashed'
+        end
+
+        card_style[:border] = '4px solid' if @game.round.can_act?(@corporation)
+
+        if selected?
+          card_style[:backgroundColor] = 'lightblue'
+          card_style[:color] = 'black'
+          card_style[:border] = '1px solid'
+        end
+
+        children = [
+          render_mine_title,
+          render_mine_holdings,
+          render_mine_machines(@corporation),
+          render_mine_status,
+]
```

Comment:
This should be indented (I wish rubocop was a little bit better at handling arrays/hashes across multiple lines...)

Reply:
Weird, I don't remember screwing up the indenting there.